src/ResourceManager/Compute/ChangeLog.md

```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Current Release
+* New parameters in New-AzureRmVMSqlServerAutoBackupConfig cmdlet to support Auto Backup for SQL Server 2016 VMs.
+* New-AzureVMSqlServer* cmdlets are renamed to New-AzureRmVMSqlServer* now. Old ones will continue to work however.
```

Comment:
@anilyela for the change log, can you talk about the new parameters in `New-AzureRmVMSqlServerAutoBackupConfig`?

```
* New parameters in ...
    - BackupSystemDbs: < brief description>
    - BackupScheduleType: <brief description>
```